daemon/containerd/image_manifest.go

```diff
@@ -237,3 +240,37 @@ func (m *ImageManifest) ReadConfig(ctx context.Context, outConfig interface{}) e
 
 	return readJSON(ctx, m.ContentStore(), configDesc, outConfig)
 }
+
+// PresentContentSize returns the size of the image's content that is present in the content store.
+func (m *ImageManifest) PresentContentSize(ctx context.Context) (int64, error) {
+	cs := m.ContentStore()
+	var size int64
+	err := c8dimages.Walk(ctx, presentChildrenHandler(cs,
+		c8dimages.HandlerFunc(func(ctx context.Context, desc ocispec.Descriptor) ([]ocispec.Descriptor, error) {
+			size += desc.Size
+			return nil, nil
+		})),
+		m.Target())
```

Comment:
OH had this file open in my IDE, and it's suggesting that the `c8dimages.HandlerFunc` conversion is redundant here (as `presentChildrenHandler` does the conversion), so this can be;


```suggestion
	err := c8dimages.Walk(ctx, presentChildrenHandler(cs, func(ctx context.Context, desc ocispec.Descriptor) ([]ocispec.Descriptor, error) {
		size += desc.Size
		return nil, nil
	}), m.Target())
```